packages/vite-plugin-cloudflare/src/module-fallback.ts

```diff
@@ -28,11 +28,7 @@ export function getResolveId(
 	});
 
 	return function resolveId(id, importer, options) {
-		const resolveMethod = options?.resolveMethod ?? 'import';
-		const resolveIdFn =
-			resolveMethod === 'import' ? esmResolveId : cjsResolveId;
-
-		return resolveIdFn(devEnvironment, id, importer);
+		throw new Error('Not implemented');
```

Comment:
The idea is that this function wouldn't actually be needed anymore

Without my patch this change would result in:
![Screenshot 2024-10-15 at 15 42 18](https://github.com/user-attachments/assets/8fdb6abb-a88b-4d56-9b74-b40d7aa8b9be)

But with my patch `zod` actually gets correctly resolved:
![Screenshot 2024-10-15 at 15 42 52](https://github.com/user-attachments/assets/e276efc0-1fe7-4657-bd12-352beac50964)

Reply:
Note: I am throwing inside `resolveId` just as an example, but this basically indicates that most if not all of the module fallback service would not be needed anymore I think